docs/sources/panels-visualizations/visualizations/status-history/index.md

```diff
@@ -149,6 +149,10 @@ Controls the opacity of state regions.
 
 {{< docs/shared lookup="visualizations/thresholds-options-2.md" source="grafana" version="<GRAFANA_VERSION>" >}}
 
+### Axis options
+
+{{< docs/shared lookup="visualizations/axis-options-size.md" source="grafana" version="<GRAFANA_VERSION>" leveloffset="+1" >}}
+
```

Comment:
```suggestion
```

Reply:
Please move this up to just before Standard options so it aligns with the order of the options in the UI. Also, make it an H2.